### PR TITLE
Ga/Ja_safe flags and mittleff clean-up

### PR DESCRIPTION
**Pull request by akabla**

### CODE REVIEW

File: test/datagen.jl [hunk 1]
```diff
@@ -76,9 +76,9 @@ function _square()
     offs = [1.5, 1.6, 1.7, 1.8, 1.9, 2.5, 2.6, 2.7, 2.8, 2.9]
     for (i,t) in enumerate(imposed.t)
         if t in ons && imposed.σ==0.0
```
Comment (moustachio-belvedere) [inline test/datagen.jl]: Did I make another mistake here? I'm wondering if it should be `imposed.\sigma[i]` in lines 78 and 80.

**Review by moustachio-belvedere — COMMENTED** [review]
Looks good to me, added two minor comments

File: src/datagen.jl [hunk 2]
```diff
@@ -90,7 +90,7 @@ time as its only argument. The original data's time signal is used.
 Normally used with a RheoTimeData generated using the `timeline` function.
 """
 function strainfunction(data::RheoTimeData, f::T) where T<:Function
-    log = data.log == nothing ? nothing : [data.log; RheoLogItem( (type=:process, funct=:strainfunction, params=(f=f,), keywords=()),
+    log = data.log === nothing ? nothing : [data.log; RheoLogItem( (type=:process, funct=:strainfunction, params=(f=f,), keywords=()),
```
[the remark this answers is not in the document]
Comment (akabla) [reply src/datagen.jl:93]: I don't know. Happy to use isnothing too; I suspect they do the same thing in the background. It's just that the "==" operator was not really appropriate here.

File: src/RHEOS.jl [hunk 3]
```diff
@@ -77,8 +81,8 @@ export AFM, Tweezers
 
 ######################################################
 # bundled dependencies from rheos-cambridge forked repos
-MittLeffLiteDir = joinpath(@__DIR__, "..", "deps", "MittLeffLite", "MittLeffLite.jl")
-include(MittLeffLiteDir)
+#MittLeffLiteDir = joinpath(@__DIR__, "..", "deps", "MittLeffLite", "MittLeffLite.jl")
```
Comment (moustachio-belvedere) [inline src/RHEOS.jl:84]: I think we can remove these lines completely? 82-85

File: src/modeldatabase.jl [hunk 4]
```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env julia
 
+#
+# Some wrapper function for external maths functions
+#
+@inline function mittleff(α,z)
```
Comment (moustachio-belvedere) [inline src/modeldatabase.jl:6]: Looks great. I think we'll need a similar one for the 3-parameter mittleff calls though

File: test/benchmarks.jl [hunk 5]
```diff
@@ -0,0 +1,88 @@
+using RHEOS
```
Comment (moustachio-belvedere) [inline test/benchmarks.jl:1]: Benchmarks are a nice addition :+1:

File: src/modeldatabase.jl (hunk 4 above)
Comment (moustachio-belvedere) [reply src/modeldatabase.jl:6]: Oh you were doing it already :+1:

**Comment by moustachio-belvedere** [comment]
Only other remnant of mittleff is probably something in the readme

**Comment by akabla** [comment]
Indeed - just remove the embedded dependency comment from the README.

codecov is very unhappy now...

**Pull request merged**